demo-extended/src/main/java/com/novoda/downloadmanager/demo/extended/DemoNotificationCustomiser.java

```diff
@@ -18,12 +18,12 @@ public DemoNotificationCustomiser(Context context) {
     }
 
     @Override
-    public void modifyQueuedOrDownloadingNotification(NotificationCompat.Builder builder, long batchId) {
+    public void modifyQueuedOrDownloadingNotification(NotificationCompat.Builder builder, com.novoda.downloadmanager.Download download) {
```

Comment:
sadly not, since this is one of the demos and there's a class called `Download` in there already :panda_face: